src/applications/ask-va/components/PrefillAlertAndTitle.jsx

```diff
@@ -3,17 +3,15 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
 
-const Title = () => (
-  <>
-    <legend className="schemaform-block-title">
-      <h3 className="vads-u-color--gray-dark vads-u-margin-top--24">
-        Your contact information
-      </h3>
-    </legend>
-  </>
+const Title = ({ title = '' }) => (
```

Comment:
Added a dynamic title prop so that this could be used in the `Your postal code` page.